pyanaconda/anaconda_logging.py

```diff
@@ -216,6 +217,13 @@ def __init__(self, write_to_journal=False):
                             minLevel=logging.NOTSET)
         self.forwardToJournal(dnf_logger)
 
+        # Create the librepo logger.
+        librepo_logger = logging.getLogger(constants.LOGGER_LIBREPO)
```

Comment:
Are the `libdnf.repo.LibrepoLog` calls in the DNF payload still valid?

Reply:
Yes, they are still necessary. Reason for that is:

> pkratoch: it still needs to be there in libdnf, so that the logs from librepo get the callback that writes them into the libdnf log (with different source), so that dnf can then process them
> pkratoch: it's a bit tangled, but this was the easiest way to unite the librepo log handling with other logs, so that the log rotation is consistent

Reply:
Updated commit message to add explanation.